<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gateway IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gateway Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="gateway_ip"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="gateway_port"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TestBox Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TestBox IP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="testBox_port"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="testBox_ip"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>MPM Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\mpmMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>MPM Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\mpmTests.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_9">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>ComCam MPM Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\comMpmMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_10">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>ComCam MPM Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\comMpmTests.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Vaccum Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\vacuumMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Vaccum Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\vacuumTests.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_11">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>ComCam Vaccum Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\comVaccumMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_12">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>ComCam Vaccum Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\comVaccumTests.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_5">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Cold Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\coldMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Cold Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\coldTests.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Cryo Monitor </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\cryoMonitor.py</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMShellCommand" name="PyDMShellCommand_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>
    A QPushButton capable of execute shell commands.
    </string>
     </property>
     <property name="text">
      <string>Cryo Tests </string>
     </property>
     <property name="allowMultipleExecutions" stdset="0">
      <bool>true</bool>
     </property>
     <property name="command" stdset="0">
      <string>python ${launcher} ${path}\\cryoTests.py</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMShellCommand</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.shell_command</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
